paas/stubs/bootstrap-stubs.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -eu
+
+echo "Bootstrapping environment: ${ENVIRONMENT?Must have set ENVIRONMENT environment variable}"
+echo "Only allowing access to test-rp web service from: ${TEST_RP_IP_SAFELIST?Must have set TEST_RP_IP_SAFELIST}"
+echo "Only allowing access to test-rp private endpoints from: ${TEST_RP_PRIVATE_IP_SAFELIST?Must have set TEST_RP_PRIVATE_IP_SAFELIST}"
+echo "Only allowing access to test-rp-msa from: ${MSA_IP_SAFELIST?Must have set MSA_IP_SAFELIST}"
+echo "Creating 'dummy' apps"
+rm -rf /tmp/paas
+mkdir -p /tmp/paas
+cd /tmp/paas
+echo "Placeholder text" > index.html
+cat <<MANIFEST > manifest.yml
+applications:
+  - name: test-rp-$ENVIRONMENT
+    routes:
+      - route: test-rp-$ENVIRONMENT.apps.internal
+      - route: test-rp-$ENVIRONMENT.cloudapps.digital
+      - route: test-rp-$ENVIRONMENT.cloudapps.digital/private
+    buildpacks:
+      - staticfile_buildpack
+
+  - name: test-rp-msa-$ENVIRONMENT
+    routes:
+      - route: test-rp-msa-$ENVIRONMENT.apps.internal
+      - route: test-rp-msa-$ENVIRONMENT.cloudapps.digital
+    buildpacks:
+      - staticfile_buildpack
+
+  - name: stub-idp-$ENVIRONMENT
+    routes:
+      - route: stub-idp-$ENVIRONMENT.cloudapps.digital
+    buildpacks:
+      - staticfile_buildpack
+MANIFEST
+cf push "test-rp-$ENVIRONMENT"
+cf push "test-rp-msa-$ENVIRONMENT"
+cf push "stub-idp-$ENVIRONMENT"
+
+# Allow test-rp to access MSA metadata
+cf add-network-policy "test-rp-$ENVIRONMENT" --destination-app "test-rp-msa-$ENVIRONMENT" --protocol tcp --port 8080
+# Allow MSA to access local matching service in test-rp
+cf add-network-policy "test-rp-msa-$ENVIRONMENT" --destination-app "test-rp-$ENVIRONMENT" --protocol tcp --port 8080
+
+git clone https://github.com/alphagov/re-paas-ip-safelist-service
+cd re-paas-ip-safelist-service
+# The templating in re-paas-ip-safelist-service doesn't really work for us so let's override it
+cat << MANIFEST > manifest.yml
```

Comment:
Can we remove a space character from the line for consistency?
```suggestion
cat <<MANIFEST > manifest.yml
```